.github/workflows/release-go-task.yml

```diff
@@ -88,11 +88,12 @@ jobs:
         run: |
           wget -q -P /tmp https://github.com/fsaintjacques/semver-tool/archive/3.2.0.zip
           unzip -p /tmp/3.2.0.zip semver-tool-3.2.0/src/semver >/tmp/semver && chmod +x /tmp/semver
-          if [[
+          if [[ \
```

Comment:
Experiments show that, unlike the `[]` alternative form of the [`test` built-in](https://pubs.opengroup.org/onlinepubs/9699919799/utilities/test.htm), the line continuation is implicit with the [Bash `[[]]` conditional compound syntax](https://www.gnu.org/software/bash/manual/bash.html#index-_005b_005b):

```bash
#!/bin/bash

if [[
  "$(
    echo "foo"
  )" != \
  ""
]]; then
  echo "test 1 evaluated as true"
else
  echo "test 1 evaluated as false"
fi

if [[
  "$(
    echo ""
  )" != \
  ""
]]; then
  echo "test 2 evaluated as true"
else
  echo "test 2 evaluated as false"
fi
```

```text
$ ./foobar.sh
test 1 evaluated as true
test 2 evaluated as false
```

However, I have never found any specification of this behavior to make it official beyond this, which is not definitively applicable to the conditional compound command:

https://www.man7.org/linux/man-pages/man1/bash.1.html#:~:text=in%20the%20list.-,Compound%20Commands,-A%20compound%20command

> In most cases a list
in a command's description may be separated from the rest of the
command by one or more newlines, and may be followed by a newline
in place of a semicolon.

So I think it is reasonable to add an explicit line continuation here, but if you are going to do that then I think it is best to "balance" it by adding another at the close of the block:

```bash
            "" \
          ]]; then
```